frontend/src/app/supportAdmin/UnarchivePatients/operations.graphql

```diff
@@ -0,0 +1,38 @@
+query GetOrganizationWithFacilities($id: ID!) {
+  organization(id: $id) {
+    externalId
+    name
+    facilities {
+      id
+      name
+    }
+  }
+}
+
+query GetPatientsByFacilityWithOrg($facilityId: ID!, $pageNumber: Int!, $pageSize: Int!, $archivedStatus: ArchivedStatus = UNARCHIVED $orgExternalId: String!) {
```

Comment:
shouldn't this be `ARCHIVED`? since we want to get patients where `isDeleted` is true

same with the query below as well

Reply:
ooo yes I can change the defaults

Reply:
oops I only realized later that you were already passing in the Archived value. nvm carry on! 😅